examples/deployments/k8s/argilla-server-deployment.yaml

```
@@ -29,6 +29,10 @@ spec:
               value: "http://elasticsearch:9200"
             - name: ARGILLA_AUTH_SECRET_KEY
               value: "CHANGE_ME"
+            - name: USERNAME
+              value: "argilla"
+            - name: PASSWORD
+              value: "12345678"
```

```suggestion
            - name: USERNAME
              value: "argilla"
            - name: PASSWORD
              value: "12345678"
```
I think we want to comment these out by default.

---

@davidberenstein1957 : Thank you for the remark. If I understand correctly, you want these environment variables to be commented like 
```
 #           - name: USERNAME
 #            value: "argilla"
 #           - name: PASSWORD
 #             value: "12345678"
```
or I should add comments to these environment variables.

If we comment, then the default owner user will not get created.

---

Yes, I think people need to be made aware of the action, because enforcing this will otherwise lead to security risks.

---

Commented the default USERNAME and PASSWORD for owner user
[Commented the USERNAME and PASSWORD field](https://github.com/argilla-io/argilla/pull/5506/commits/811fb3df1f58ff90a06ec0d243f2e91eec9204db#diff-75693acb7abb4409fa1d7c8fe68045e41a1731d228bac6cf0f154a2f5c94401eL32)